src/core/Entry.cpp

```diff
@@ -1277,11 +1277,11 @@ void Entry::setGroup(Group* group, bool trackPrevious)
         }
     }
 
+    QObject::setParent(group);
```

Comment:
This line shouldn't affect fdosecrets, it is just assigning Qt object hierarchy so when the group object is deleted, the entry object is also deleted.

Reply:
Sure. I just found it to be the wrong order and therefore put it into a separate commit. I can drop the commit if this change is not desired, though.

Reply:
Technically, the order of the calls doesn't matter